frontend/src/constants.js

```diff
@@ -58,13 +58,15 @@ export const VIEW_ALL = "VIEW_ALL";
 export const ASSIGN_REVIEWERS = "ASSIGN_REVIEWERS";
 export const REVIEW_CARDS = "REVIEW_CARDS";
 export const TRUSTED_REVIEWER = "TRUSTED_REVIEWER";
+export const VIEW_ONLY = "VIEW_ONLY";
 
 export const TEAM_PERMISSIONS = [
   MANAGE_CARDS,
   VIEW_ALL,
   ASSIGN_REVIEWERS,
   REVIEW_CARDS,
   TRUSTED_REVIEWER,
+  VIEW_ONLY
```

Comment:
Permissions need to be implemented on the backend. VIEW_ONLY doesn't exist anywhere so this definately wont work.

You might as well have added a permission called `CAN_HAZ_SANDWICH`, it would have had the same effect. The frontend is a representation of what the backend knows. If you write logic around things that dont mean anything then the logic is meaningless.